src/frequenz/sdk/microgrid/__init__.py

```diff
@@ -74,16 +74,31 @@
 
 ## Producers: PV Arrays, CHP
 
-The total {{glossary("pv", "PV")}} power production in a microgrid can be streamed
-through [`pv_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.pv_power] , and
-similarly the total CHP production in a site can be streamed through
-[`chp_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.chp_power].  And total
-producer power is available through
-[`producer_power`][frequenz.sdk.timeseries.producer.Producer.power].
+The total CHP production in a site can be streamed through
+[`chp_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.chp_power].  PV Power
+is available through the PV pool described below.  And total producer power is available
+through [`producer_power`][frequenz.sdk.timeseries.producer.Producer.power].
```

Comment:
Unrelated to this PR, but Isn't this `producer.power` now?

Reply:
True, updated.